packages/generator/src/index.ts

```diff
@@ -32,14 +33,25 @@ export function main(): void {
   }
 }
 
+class GeneratorFileBuilder extends FileBuilder {
```

Comment:
I'd be ok to cut this, especially if the only difference is the "Generated by ..." message.